### PR TITLE
Desktop style appointment page

### PR DESCRIPTION
**Pull request by stephanie-saretsky**
mobile:
Screen Shot 2022-11-03 at 10 26 21 AM
Screen Shot 2022-11-03 at 10 26 33 AM
Screen Shot 2022-11-03 at 10 26 50 AM

desktop:
Screen Shot 2022-11-03 at 10 27 29 AM
Screen Shot 2022-11-03 at 10 27 41 AM
Screen Shot 2022-11-03 at 10 27 54 AM

### Ticket

[ch8990]

### Accomplishments

- [x] Common page container
- [x] Common title 
- [x] Added these to chat page only if ff is enabled and is not mobile

### Feature flag

- [x] If there is a way to toggle the work in this PR via a feature flag, I have added one
- [x] The flag has been created in [LaunchDarkly](https://app.launchdarkly.com/kariko/dev/features) for all environments

### CODE REVIEW

**Comment by LUMacKenn** [comment]
I wasn't able to get this working despite turning on the feature flag for my account (both local and dev), and in addition the logo seems to have broken in the header
Screen Shot 2022-11-01 at 5 28 56 PM

File: src/features/conversations/pages/Conversations.tsx [hunk 1]
```diff
@@ -1,24 +1,58 @@
...
-      />
-      <ConversationsContainer />
+    <Box sx={{ display: "flex", flexDirection: "column", height: "100%", width: "100%" }}>
+      {!isMobile && desktopEnabled ? (
+        <div style={{ width: "100%" }}>
+          <DesktopTitle
+            title="Chat"
+            button={
+              <span style={{ cursor: "pointer" }} onClick={() => navigate("/chat/create")}>
+                <PlusIcon
```
Comment (lvancraen) [inline src/features/conversations/pages/Conversations.tsx:26]: Curious to know if you even need the <span /> here. Can you add the onClick to the PlusIcon directly?

File: src/features/conversations/pages/Conversations.tsx [hunk 2]
```diff
@@ -1,24 +1,58 @@
...
+            <ConversationsContainer />
+          </PageContainer>
+        </div>
+      ) : (
+        // replace with just isMobile when flag is no longer needed
+        <>
+          <MobileTitle
+            title="Chat"
+            button={
+              <span style={{ cursor: "pointer" }} onClick={() => navigate("/chat/create")}>
```
Comment (lvancraen) [inline src/features/conversations/pages/Conversations.tsx:47]: Same question as above

File: src/features/conversations/pages/Conversations.tsx (hunk 1 above)
Comment (stephanie-saretsky) [reply src/features/conversations/pages/Conversations.tsx:26]: Yeah that's a good point! It was already in the code so I just copied it. Will test if that will work.

**Comment by stephanie-saretsky** [comment]
> I wasn't able to get this working despite turning on the feature flag for my account (both local and dev), and in addition the logo seems to have broken in the header Screen Shot 2022-11-01 at 5 28 56 PM

@LUMacKenn what did you have in your env.local for the logos? We have been updating the env vars so it might not be compatible. You don't show up as a user in launch darkly, so that is why the feature flag is not working. You need to ensure you're using the correct LD env var.

**Pull request merged**